client/mobilizations/widgets/components/input-tag.js

```diff
@@ -26,8 +26,8 @@ class InputTag extends Component {
     // Linux    : ctrl + enter
     //
     const mac = os.isMac() && keycode(e) === 'enter' && e.nativeEvent.metaKey
-    const windows = os.isWindows() && keycode(e) === 'enter' && e.nativeEvent.ctrlKey
-    const linux = os.isLinux() && keycode(e) === 'enter' && e.nativeEvent.ctrlKey
+    const windows = os.isWindows() && keycode(e) === 'enter' && e.ctrlKey
+    const linux = (os.isLinux() || os.isUnix()) && keycode(e) === 'enter' && e.ctrlKey
```

Comment:
@lpirola I get the OS detection from this Stack Overflow answer: https://stackoverflow.com/a/11219680

When I access the BONDE to test it on my development environment using a VM of linux, the `window.navigator.appVersion` was returning as **UNIX**... I really do not know for sure if, some version of MacOS will return as **UNIX** on `window.navigator.appVersion`..

Do you have any suggestion to solve this using a less ugly solution?